hardware/rtc.py

```diff
@@ -19,17 +19,18 @@
 '''
 import logging
 import re
-from subprocess import check_output
+import subprocess
 
 LOG = logging.getLogger('hardware.rtc')
 
 
 def get_rtc():
     cmd = ['timedatectl', 'status', '--no-pager']
     try:
-        stdout = check_output(cmd).decode("utf-8")
-    except OSError:
-        LOG.warning('Unable to determine RTC timezone (no timedatectl)')
+        stdout = subprocess.check_output(cmd).decode("utf-8")
+    except subprocess.CalledProcessError as excpt:
+        LOG.warning('Unable to determine RTC timezone (no timedatectl) - %s',
```

Comment:
"no timedatectl" actually implies `OSError`, it's what is raised when the executable is not found. I'd suggest we keep both OSError and CalledProcessError to not regress in this case.

Reply:
Restored OSError handling.